backend/src/database/entities/training.entity.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('training')
+export class TrainingEntity {
+  @PrimaryGeneratedColumn('increment')
+  id: string;
+
+  @Column({ name: 'name', type: 'varchar', length: 100 })
```

Comment:
Will we add a new 'training' type every time a user is added (or their training updated in future)? 

Or will this be a static table with predefined trainings that we can map to the resource?

Reply:
The latter! Personnel defines this
```
@ManyToMany(() => TrainingEntity)
  @JoinTable({ name: 'personnel_training' })
  trainings: TrainingEntity[];
```